drivers/scheduler/k8s/k8s.go

```diff
@@ -1461,6 +1465,49 @@ func (k *k8s) StartSchedOnNode(n node.Node) error {
 	return nil
 }
 
+func (k *k8s) CreateCRDObjects(pathCRDSpec string) error {
```

Comment:
This is an anti-pattern with how other things are done currently. Can we do the parsing at the same place other spec files are parsed and this thing just takes in a context?

Reply:
Can you share where in stork integration test are we creating the file?

As as when possible, I want to keep the same pattern for all interfaces. Don't want one interface to take a file and other take pre-loaded contexts.

Reply:
That is not part of stork integration test yet, because creation of cluster pair depend on this PR for getting cluster token. 
This is my tryout branch if you want to have look -  https://github.com/libopenstorage/stork/compare/rs-ClusterMigration?expand=1#diff-6e48c877f1215e1e1f64e38abe60a903R33

Reply:
Corrected to take context and use those for applying spec file. Since we have ```storkctl ``` command line to generate spec file we can use that as prerequisite to create and dump clusterpair spec file